ui/pages/swaps/prepare-swap-page/review-quote.js

```diff
@@ -0,0 +1,1309 @@
+import React, {
```

Comment:
Most of the logic here is reused from the View Quote page: https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/swaps/view-quote/view-quote.js